services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/db/repo/PortRedisRepository.java

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.dataplane.db.repo;
+
+import com.futurewei.alcor.dataplane.logging.Logger;
+import com.futurewei.alcor.dataplane.logging.LoggerFactory;
+import com.futurewei.alcor.dataplane.model.PortState;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.redis.core.HashOperations;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import java.util.Map;
+import java.util.logging.Level;
+
+@Repository
+public class PortRedisRepository implements ICacheRepository<PortState> {
```

Comment:
Don't think we will need Port, subnet, vpc repositories here.

Reply:
fixed in 136f4b5